pkg/controllers/provisioningv2/rke2/bootstrap/controller_test.go

```diff
@@ -0,0 +1,274 @@
+package bootstrap
```

Comment:
I am not sure how I feel about this. If my knowledge is correct, this will be the only unit tests in Rancher that mocks any controller interfaces.

I'm not saying that it should be removed, but I am certainly not opposed to removing it.

Reply:
We have been instructed and requested to write unit tests for code that we are committing to Rancher. This unit test ensures that Windows functionality will be verified alongside Linux.